prototypes/items.lua

```diff
@@ -270,6 +272,8 @@ data:extend({
   { -- Napalm
     type = "ammo",
     name = "napalm",
+    -- added due to 2.0 modding API changes

```

Comment:
We should remove the "added due to 2.0 modding API changes" comments, as the git commit has indicated the changes.